cdap-app-fabric/src/main/java/io/cdap/cdap/gateway/handlers/AppLifecycleHttpHandler.java

```diff
@@ -495,7 +497,7 @@ public void updateApp(FullHttpRequest request, HttpResponder responder,
   }
 
   /**
-   * upgrades an existing application.
+   * upgrades the existing application.
    */
   @POST
   @Path("/apps/{app-id}/upgrade")
```

Comment:
@chtyim @tivv I am not including /upgrade or /update api changes as part of this PR - we can confirm expected behavior offline after this merge. JIRA for tracking https://cdap.atlassian.net/browse/CDAP-19782